Terminal.Gui/Views/TextField.cs

```diff
@@ -732,7 +758,10 @@ void MoveWordLeft ()
 			Adjust ();
 		}
 
-		void RedoChanges ()
+		/// <summary>
+		/// Redoes the latest changes.
+		/// </summary>
+		public void Redo ()
 		{
 			if (ReadOnly)
 				return;
```

Comment:
Still missing braces.

Reply:
Sorry I only did in the TextView.cs. Fixed with the https://github.com/gui-cs/Terminal.Gui/pull/2682/commits/f9846b42bd137c6b806323b6b26258e22b9ff55c.